.github/workflows/ci-fast-tooling.yml

```diff
@@ -0,0 +1,41 @@
+name: CI - FAST Tooling
+
+on:
+  push:
+    branches: 
+      - release
+  pull_request:
+    branches: 
+      - release 
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+    
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Cache multiple paths
+      uses: actions/cache@v2
+      with:
+        path: |
+          ~/cache
+          !~/cache/exclude
+        key: ${{ runner.os }}-${{ hashFiles('**/lockfiles') }}
```

Comment:
Where do "lockfiles" come from? We have a package-lock.json since we are using only npm 7